client/src/components/v3/status/WMStatus.tsx

```diff
@@ -22,6 +22,91 @@ export default function WMStatus(props: WMStatusProps) {
     </span>
   );
 
+  function convertToSentenceCase(str: String) {
```

Comment:
This is actually converting to `Title Case`, since each word is capitalised from the camel case input. Maybe you want to convert the entire string to lower case before altering the first character? Neat regex solution by the way!